front/lib/models/assistant/actions/dust_app_run.ts

```diff
@@ -157,6 +158,10 @@ AgentDustAppRunAction.init(
       type: DataTypes.JSONB,
       allowNull: true,
     },
+    step: {
+      type: DataTypes.INTEGER,
+      allowNull: true,
```

Comment:
@lasryaric Let's make sure to remove this switch this allowNull now that this is shipped :pray: